### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by koko0314** [GaiZhenbiao/ChuanhuChatGPT#870] [Q&A]
Error
☹️发生了错误：{ “error”: { “message”: “You exceeded your current quota, please check your plan and billing details.”, “type”: “insufficient_quota”, “param”: null, “code”: “insufficient_quota” }}

**Comment by GaiZhenbiao** [comment]
“您已超过当前配额，请检查您的计划和账单详细信息。”

你的赠金用完了。
